tests/testthat/test-sma-sim.R

```diff
@@ -133,6 +133,20 @@ test_that("sacramento simulation is the same in R and C", {
   }
 })
 
+test_that("hbv simulation is the same in R and C", {
+  set.seed(0)
+  hbvDATA <- cbind(P = P, E = E, T = E - 15)
```

Comment:
What's the motivation behind T = E -15?

Reply:
E appeared to be temperature data, so I used E-15 so that temperatures went below 0 to invoke the snow routine. It appears I did not think of dealing with E being temperature and not PET for the model. Got a bit lazy.

I'll check out the patrick tests

Reply:
Ah. 
Let's just use the existing data for now, but with DATATE having E in mm rather than temperature, even if this means test coverage won't be 100%.
Later:
Let's add a separate test dataset for snow #163 
Let's also separately decide how to handle T vs E #32